xsrc/test/scripts/refactor/fix-imports.scala

```diff
@@ -0,0 +1,184 @@
+import java.io.BufferedWriter
+import java.io.FileWriter
+import java.io.File
+import scala.collection.mutable.ListBuffer
+import scala.collection.mutable.Map
+import scala.io.Source
+
+object FixImports extends App {
+  val inline = true
+  val importMap = Map(
+    "daffodil-cli" -> "cli",
+    "daffodil-core" -> "core",
+    "daffodil-io" -> "io",
+    "daffodil-lib" -> "lib",
+    "daffodil-macro-lib" -> "lib",
```

Comment:
This might want to be `org.apache.daffodil.macros`? This package isn't actually published anywhere so it probably doesn't really matter, but it might help to make it more clear in the codebase where macros are actually used if they are in a separate namespace.

Reply:
I actually did that at first but the code that uses them expect the macros to be in the same package and they are actually split across multiple packages with `lib` just being one of them. This might require some changes to the build.sbt. I think for this refactoring we should keep the functionality the same, and if we want to change this, it could be done in a separate PR.